main.go

```diff
@@ -14,6 +14,8 @@ import (
 
 var version = "unknown"
 
+const applySleepSeconds = 1
```

Comment:
I don't know if I want to give people the ability to tune this. Setting it higher does seem like a good idea in my mind.